src/models/flinkStatement.ts

```diff
@@ -124,6 +125,34 @@ export class FlinkStatementTraits {
   }
 }
 
+export class FlinkStatementSpec {
```

Comment:
Need to more properly model the a flink statement's 'spec', describing both the sql statement and the compute pool (was modeled prior, but skimpily not within the spec).